DB/mysqli.php

```diff
@@ -498,7 +498,11 @@ function fetchInto($result, &$arr, $fetchmode, $rownum = null)
      */
     function freeResult($result)
     {
-        return is_resource($result) ? mysqli_free_result($result) : false;
```

Comment:
I didn't actually find the core file - the link above doesn't have that line in it